LongFieldMapperTests test testFetchCoerced failing

LongFieldMapperTests testFetchCoerced failing

**Build Scans:**
- [Failed CI Run](https://gradle-enterprise.elastic.co/s/laeomhqjtvjf4/tests/:server:test/org.elasticsearch.index.mapper.LongFieldMapperTests/testFetchCoerced)

**Reproduction Line:**
```

./gradlew ":server:test" --tests "org.elasticsearch.index.mapper.LongFieldMapperTests.testFetchCoerced" -Dtests.seed=1D9DCAF60052564D -Dtests.locale=tk-Latn-TM -Dtests.timezone=Pacific/Guadalcanal -Druntime.java=24
```

**Failure Message:**
```
java.lang.AssertionError: fetching 3.783147882954537E18
Expected: iterable with items [<3783147882954536960L>] in any order
but: not matched: <3783147882954537000L>
```

**Note:**
This issue was created from stale `AwaitsFix` test. See [#132306](https://github.com/elastic/elasticsearch/pull/132306) for more details.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -474,9 +474,6 @@ tests:
 - class: org.elasticsearch.upgrades.TransformSurvivesUpgradeIT
   method: testTransformRollingUpgrade
   issue: https://github.com/elastic/elasticsearch/issues/132892
-- class: org.elasticsearch.index.mapper.LongFieldMapperTests
-  method: testFetchCoerced
-  issue: https://github.com/elastic/elasticsearch/issues/132893
 - class: org.elasticsearch.xpack.eql.planner.QueryTranslatorTests
   method: testMatchOptimization
   issue: https://github.com/elastic/elasticsearch/issues/132894

--- a/server/src/test/java/org/elasticsearch/index/mapper/LongFieldMapperTests.java
+++ b/server/src/test/java/org/elasticsearch/index/mapper/LongFieldMapperTests.java
@@ -118,10 +118,6 @@ public class LongFieldMapperTests extends WholeNumberFieldMapperTests {
         return randomDoubleBetween(-MAX_SAFE_LONG_FOR_DOUBLE, MAX_SAFE_LONG_FOR_DOUBLE, true);
     }
 
-    public void testFetchCoerced() throws IOException {
-        assertFetch(randomFetchTestMapper(), "field", 3.783147882954537E18, randomFetchTestFormat());
-    }
-
     protected IngestScriptSupport ingestScriptSupport() {
         return new IngestScriptSupport() {
             @Override

